controller_manager/src/controller_manager.cpp

```diff
@@ -2558,7 +2558,7 @@ void ControllerManager::controller_activity_diagnostic_callback(
   }
   else
   {
-    stat.summary(diagnostic_msgs::msg::DiagnosticStatus::ERROR, "Not all controllers are active");
+    stat.summary(diagnostic_msgs::msg::DiagnosticStatus::WARN, "Not all controllers are active");
```

Comment:
I wouldn't mark it as `WARN` as it could be intended or you just deactivated a controller to activate a different one. IMO, it doesn't make sense to be a WARNING or an ERROR.

Reply:
Sounds good to me.